test/unit/verify/test_models.py

```diff
@@ -29,29 +32,43 @@ class TestVerificationMaterials:
     def test_rekor_entry_inconsistent_cve_2022_36056(
         self, signing_materials, signing_bundle
     ):
-        a_materials = signing_materials("a.txt")
-        offline_rekor_materials = signing_bundle("bundle.txt")
+        (_, a_materials) = signing_materials("a.txt")
+        (file, offline_rekor_materials) = signing_bundle("bundle.txt")
+
+        with file.open(mode="rb", buffering=0) as input_:
```

Comment:
That works, although to go one step further: I think `signing_materials`/`signing_bundle` should probably just return `bytes`/`Hashed` directly, since the plan is to have both `sign` and `verify` take those types rather than do their own I/O 🙂